config/config.go

```diff
@@ -224,6 +225,10 @@ type Config struct {
 	StorageConfig     StorageConfig   `yaml:"storage,omitempty"`
 	TracingConfig     TracingConfig   `yaml:"tracing,omitempty"`
 
+	// Secret management:
+	kubernetes.ClientConfig  `yaml:"kubernetes_sp_config,omitempty"`
+	kubernetes.SecretConfigs `yaml:"secrets,omitempty"`
```

Comment:
non-blocker for fork: This should be generic secret config, not kubernetes specific one I guess?

Reply:
Intentionally different. I want us to be different from upstream in case the final design changes.